### DISCUSSION TITLE
Javascript for Magento_ReCaptchaCustomer not executing when using recaptcha on customer login / forgot password with Breeze Evolution theme

### DISCUSSION THREAD

**Discussion by neilblackwood** [breezefront/community#225] [1. Help] [answered]
Hi all,

I've enabled the Magento built recaptcha checkbox on the signin / forgotten password page and the default behaviour on page load, is for the submit button to be loaded disabled. The javascript should execute after page load and the sign in button should be updated from disabled to enabled, but In combination with the Breeze Evolution theme, nothing happens - there is also no error in the javascript console logs - I'm wondering if it's something to do with the Breeze functionality being enabled because when I disable the Breeze functionality the javascript appears to be executed correctly. Is there a workaround or instructions on how to fix this problem?

Many thanks

**Comment by vovayatsyuk** [comment, marked as answer]
Hello,

Looks like the bug was fixed in 2.11: https://github.com/breezefront/module-breeze/commit/688b53ce7a5b668910b1cfc57243127f040323bf

**Reply by neilblackwood** [reply]
That's great thanks, all sorted!
